spec/model/model_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
```

Comment:
- [ ] Kindly add tests for the Like and Comment models. All models should be tested. Remember to add tests for the methods within those models.

- [ ] **[OPTIONAL]** Kindly rename this test file to something like `post_spec.rb` as that is a more descriptive name for the tests it contains.